commit f6861b82040b34ab5771d7532cd4c12237ba1e89 (refs/changes/61/4961/2)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2018-10-16T18:55:36+00:00 (12 months ago)
    
    TRITON-755 Want ability to filter CMON discovery results by tag
    
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/package.json b/package.json
index e63b5fc..a71f7ff 100644
--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "restify-errors": "4.3.0",
     "sdc-scripts": "git+https://github.com/joyent/sdc-scripts.git#deefaef587ed3bee2706cb6e53ee3468e682932e",
     "sprintf-js": "1.0.3",
-    "triton-tags": "^1.1.2",
+    "triton-tags": "1.2.2",
     "uuid": "2.0.1",
     "vasync": "1.6.3"
   }
